src/Cubes/Cubes.jl

```diff
@@ -74,7 +74,7 @@ finalizer function for CleanMe struct. The main process removes all directories/
 function clean(c::CleanMe)
     if !c.persist && myid() == 1
         if !isdir(c.path) && !isfile(c.path)
-            @warn "Cube directory $(c.path) does not exist. Can not clean"
```

Comment:
why? why removing this is necessary ?

Reply:
Hi Lazaro,

I'm running into some warnings, but this change didn't solve the issue. @meggart could you please confirm if this is still necessary?

Reply:
With new julia updates it is not possible anymore to print anything to screen during running a finalizer, so an error message will be printed saying "Task switch not allowed during finalizer" or similar, which can be very confusing so I would prefer not to throw the warning.